docs/fastdds/logging/disable_module.rst

```diff
@@ -30,11 +29,13 @@ However, it is possible to fully disable each macro (and therefore each verbosit
 
 Applying either of the previously described methods will set the macro to be empty at configuration time, thus allowing
 the compiler to optimize the call out.
-|logInfo| is a special case worth mentioning; |logInfo| is only active is ``CMAKE_BUILD_TYPE`` is set to ``Debug``, or
-if ``INTERNAL_DEBUG`` is set to ``ON``.
 This is done so that all the debugging messages present on the library are optimized out at build time if not building
 for debugging purposes, thus preventing them to impact performance.
 
+``INTERNAL_DEBUG`` CMake option activates log macros compilation, so the arguments of the macros are compiled and
+executed.
+But it does not activate the log messages (it does not force log to write the messages in the macros).
+
```

Comment:
```suggestion
``INTERNAL_DEBUG`` CMake option activates log macros compilation, so that the arguments of the macros are compiled and
executed.
However it does not activate the log messages, i.e. the log does not write the messages in the macros.

```

Reply:
:+1: 
Other error in INTERNAL_DEBUG explanation has been corrected.